pkgs/development/tools/pre-commit/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, pythonPackages, pew }:
+with pythonPackages; buildPythonApplication rec {
+  name = "${pname}-${version}";
+  pname = "pre_commit";
+  version = "1.10.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1kn8h9k9ca330m5n7r4cvxp679y3sc95m1x23a3qhzgam09n7jwr";
+  };
+
+  propagatedBuildInputs = with pythonPackages; [
+    aspy.yaml
+    cached-property
+    cfgv
+    identify
+    nodeenv
+    six
+    toml
+    virtualenv
+  ];
+
+  meta = with stdenv.lib; {
+    description = "A framework for managing and maintaining multi-language pre-commit hooks.";
+    homepage = https://pre-commit.com/;
+    license = licenses.mit;
+    platforms = platforms.all;
```

Comment:
drop platforms, is already set

Reply:
Oh, buildPythonApplication does that?  I dropped it.